main/getting-started/intro-zoe.md

```diff
@@ -158,16 +157,14 @@ proposal:
 
 Proposals must use Keywords, which are capitalized ASCII keys. Here,
```

Comment:
@erights This differs from the definition we settled on in [#638](https://github.com/Agoric/documentation/pull/638):
> A keyword is a property name string that starts with an upper case letter and is a valid
[identifier](https://developer.mozilla.org/en-US/docs/Glossary/Identifier).

Is this ASCII constraint accurate?

Reply:
My mistake. I just looked, and the ascii constraint is accurate.

Reply:
No worries, I just added the fix to this PR: https://github.com/Agoric/documentation/pull/651/commits/77a6067a43e6299e1c4e6530df080862ffb808ff